api/access/server.go

```diff
@@ -185,3 +242,24 @@ func (s *Server) GetNetworkParameters(ctx context.Context, in *access.GetNetwork
 func (s *Server) GetLatestProtocolStateSnapshot(ctx context.Context, in *access.GetLatestProtocolStateSnapshotRequest) (*access.ProtocolStateSnapshotResponse, error) {
 	return nil, errors.New("not implemented")
 }
+
+func eventsFromBlock(header *flow.Header, ee []flow.Event, blockID flow.Identifier, height uint64) (*access.EventsResponse_Result, error) {
+	timestamp, err := ptypes.TimestampProto(header.Timestamp)
+	if err != nil {
+		return nil, err
```

Comment:
Even if it's the only error here, this should be where you wrap it. We might have other errors later.